client/index.html

```diff
@@ -274,7 +274,7 @@ <h1 class="title">Settings</h1>
 								<label for="theme-select" class="sr-only">Theme</label>
 								<select id="theme-select" name="theme" class="input">
 									{{#each themes}}
-										<option value="{{filename}}">
+										<option value="{{url}}">
```

Comment:
This breaks backwards compatibility, currently the value is just the theme name like `example`, but this PR changes it to a URL. I assume you changed to url because local themes do not have `plugins/` in it, but maybe a better fix would be to load local themes just the same way as plugins?